tests/codegen/pointer_inc_var.cpp

```diff
@@ -0,0 +1,14 @@
+#include "common.h"
+
+namespace bpftrace {
+namespace test {
+namespace codegen {
+
+TEST(codegen, pointer_inc_var)
+{
+  test("kprobe:f { $v = (int16*)1000; $v++ }", NAME);
```

Comment:
What about doing something like `$x = $v++` to test that the original value of `$v` is assigned? Also, a similar runtime test would be nice.

Reply:
ah yeah won't hurt to have